internal/component/kafka/consumer.go

```diff
@@ -190,62 +197,40 @@ func (consumer *consumer) Setup(sarama.ConsumerGroupSession) error {
 	return nil
 }
 
-// AddTopicHandler adds a topic handler
-func (k *Kafka) AddTopicHandler(topic string, handler EventHandler) {
-	k.subscribeLock.Lock()
-	k.subscribeTopics[topic] = handler
-	k.subscribeLock.Unlock()
-}
-
-// RemoveTopicHandler removes a topic handler
-func (k *Kafka) RemoveTopicHandler(topic string) {
-	k.subscribeLock.Lock()
-	delete(k.subscribeTopics, topic)
-	k.subscribeLock.Unlock()
-}
-
-// AddTopicBulkHandler adds a bulk handler and configuration for a topic
-func (k *Kafka) AddTopicBulkHandler(topic string, handlerConfig BulkSubscriptionHandlerConfig) {
+// AddTopicHandler adds a bulk handler and configuration for a topic
+func (k *Kafka) AddTopicHandler(topic string, handlerConfig SubscriptionHandlerConfig) {
 	k.bulkSubscribeLock.Lock()
```

Comment:
Do we need a `bulkSubscribeLock`, or could `subscribeLock` be used instead? Since you're modifying the same `subscribeTopics` map

Reply:
My bad, I should have removed it. Done.